dev/create-release/release-tag.sh

```diff
@@ -42,7 +42,7 @@ EOF
 set -e
 set -o pipefail
 
-if [[ $@ == *"help"* ]]; then
+if [[ "$*" == *"help"* ]]; then
```

Comment:
SC2199 (error): Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).